dao-utils/src/bytes.rs

```diff
@@ -1 +0,0 @@
-
```

Comment:
empty file?

Reply:
Nope, the file had been empty, I just removed it.